### PR TITLE
feat: add CollectorScheduleSection edit mode markups

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

SSIA

https://github.com/cloudforet-io/console/assets/26986739/b1f7dfed-3780-4c48-853b-f040ea9719a9

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorServiceAccountsSection.vue [hunk 1]
```diff
@@ -45,7 +45,7 @@ interface SecretModel {
     created_at: string;
 }
 const props = defineProps<{
-    providers: string[]|null
+    providers?: string[]|null
```
Comment (piggggggggy) [inline apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorServiceAccountsSection.vue:48]: 👍🥲

File: apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorScheduleForm.vue [hunk 2]
```diff
@@ -0,0 +1,139 @@
+<template>
```
Comment (sulmoJ) [inline apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorScheduleForm.vue:1]: Question: How can we take the folder structure of the components used as this form?

**Review by sulmoJ — APPROVED** [review]
LGTM!

File: apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorScheduleForm.vue [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+const props = defineProps<{
+    editHours: boolean;
+}>();
+
+const hoursMatrix = range(24);
+const state = reactive({
+    timezone: computed(() => store.state.user.timezone),
+    isAutoSchedule: true,
+    // TODO: remove temporary data
+    selectedHours: { 1: dayjs() } as ScheduleHours,
```
Comment (piggggggggy) [inline apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorScheduleForm.vue:70]: 👀

**Review by piggggggggy — APPROVED** [review]
LGTM! I want to know `ScheduleHours` usage.

File: apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorScheduleForm.vue (hunk 2 above)
Comment (WANZARGEN) [reply apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorScheduleForm.vue:1]: Yeah.. We need to discuss it. I think we need to change the folder structure soon. 🥲

**Pull request merged**